CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ CHANGELOG
 - Upgrade Slurm to version 21.08.5.
 - Disable EC2 ImageBuilder enhanced image metadata when building ParallelCluster custom images.
 - Disable packages update at instance launch time on Amazon Linux 2.
+- If security groups are specified for some head node and computes but not all, the security groups created by ParallelCluster allows inbound connections from the custom security groups.
```

Comment:
I'd move this to enhancements. I'd write something like:

Improve Security Groups created within the cluster allowing inbound connections from custom security groups when `SecurityGroups` parameter is specified for Head Node or Queue, but not both.`

In general we should start the sentence with an action (Add/Improve/Disable, etc).

Reply:
Done